main.py

```diff
@@ -63,22 +80,45 @@ def process_competitions(size: int) -> list:
         if item.endswith(".csv"):
             file_path = os.path.join(CSV_PATH, item)
             df_comp = pd.read_csv(file_path)
-            df_comp = df_comp.filter(["TeamName", "Score"])
-            df_comp.columns = ["Name", "Score-"+item]
+            df_comp = df_comp.filter(["TeamName", "Score", "SubmissionCount"])
+            df_comp.columns = ["Name", "Score-" + item, "Count-" + item]
             df = df_comp if df.empty  else pd.merge(df, df_comp, on = ["Name"], how = "outer")
+    
+    # create filters
+    score_filter = []
+    attempt_filter = []
+    for item in os.listdir(CSV_PATH):
+        if item.endswith(".csv"):
+            score_filter.append("Score-" + item)
+            attempt_filter.append("Count-" + item)
 
-    # compute combined score (placeholder)
-    df["Score"] = df.drop(columns=["Name"]).sum(axis=1, skipna=True)
-    df["Score"] = df["Score"].round(5)
-    df.sort_values(by=['Score'], inplace=True, ascending=False)
-    df = df.reset_index()
+    # aggregate attempts and compute score
+    df["Attempts"] = df.filter(attempt_filter).sum(axis = 1, skipna = True)
+    df["Score"] = score_metric(df.filter(score_filter))
+
+    # sort 
+    df.sort_values(by = ['Score'], inplace = True, ascending = False)
+    df.reset_index(inplace = True, drop = True)
 
     # Format to array
-    data = []
+    curr_standings = []
     for _, row in df.head(size).iterrows():
-        data.append({"Team": row["Name"], "Score": row["Score"]})
-
-    return data
+        curr_standings.append({"Team": row["Name"], "Score": row["Score"], "Attempts Remaining": (maxAttempts - row["Attempts"]), "Delta": "-"})
```

Comment:
Can u change the field names to the following:

```
Team => team
Score => score
Attempts Remaining => attemptsLeft
Delta => delta
```

Reply:
I have just run the code and it looks like the code above used to calculate the `Attempts Remaining` is able to generate negative values: 

Examples: 
![image](https://github.com/theDS3/Datathon-Leaderboard/assets/32755895/27c774e0-783c-4e83-80db-7ce863fbe369)
![image](https://github.com/theDS3/Datathon-Leaderboard/assets/32755895/44ecf1b8-87d4-45f7-b412-c18d532c87ce)

Reply:
Yeah the negative values are because we are currently using random competitions, where the submission limit isn't enforced (or is larger than what ours will be).

Reply:
I'll change the field names